islandora_pathauto.module

```diff
@@ -132,6 +133,7 @@ function islandora_pathauto_islandora_object_alter($original, $context) {
       $object->label = $context['params']['label'];
     }
   }
+  //I'm thinking that line 136 is useless? Doesn't even update the alias
```

Comment:
This call to `islandora_pathauto_create_alias()` would be important if the object was altered by something other than the properties form.  Consider, for example, a batch script to update all object labels.  The batch script would make a change to the object, which would trigger the call of the hook_islandora_object_alter().  This module then gets the opportunity to examine the changes and determine if the alias needs to be updated.